workspaces/redhat-argocd/plugins/argocd-backend/config.d.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2025 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export interface Config {
+  /** ArgoCD Configurations for the Red Hat ArgoCD backend plugin */
+  argocd?: {
+    /**
+     * @visibility backend
```

Comment:
The default is backend so you don't need this but it does help provide extra clarity 👍